packages/cashier/src/Components/recent-transaction/recent-transaction.jsx

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { ButtonLink, Text, Icon } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { epochToMoment } from '@deriv/shared';
+import { connect } from 'Stores/connect';
+import { getStatus } from '../../Constants/transaction-status';
+
+const RecentTransaction = ({ crypto_transactions, currency, onMount }) => {
+    React.useEffect(() => {
+        onMount();
+    }, []);
```

Comment:
I believe you can use `onMount` as the dependency array of this `useEffect`, if I'm wrong please add relevant comment for disabling the eslint warning.